Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -526,99 +538,68 @@ private void EnsureMixedRealityRequirements()
 
 #region MonoBehaviour Implementation
 
-        private static MixedRealityToolkit instance;
+        private static MixedRealityToolkit activeInstance;
         private static bool newInstanceBeingInitialized = false;
 
+        /// <summary>
+        /// Lock property for the Mixed Reality Toolkit to prevent reinitialization
+        /// </summary>
+        private static readonly object initializedLock = new object();
+
         /// <summary>
         /// Returns the Singleton instance of the classes type.
         /// If no instance is found, then we search for an instance in the scene.
-        /// If more than one instance is found, we log an error and no instance is returned.
+        /// If more than one instance is found, only the first instance is activated. All others are deactivated.
         /// </summary>
         public static MixedRealityToolkit Instance
         {
             get
             {
-                if (IsInitialized)
+                if (activeInstance != null)
                 {
-                    return instance;
-                }
-
-                if (Application.isPlaying && !searchForInstance)
-                {
-                    return null;
+                    return activeInstance;
                 }
 
-
                 var objects = FindObjectsOfType<MixedRealityToolkit>();
```

Comment:
I think we should remove this check. 
If there is no instance in the scene, this check will run each time we do a null check against the instance, and no instance will ever be added.